scalding-hadoop-test/src/test/scala/com/twitter/scalding/platform/PlatformTest.scala

```diff
@@ -701,6 +717,13 @@ class PlatformTest extends WordSpec with Matchers with HadoopSharedPlatformTest
     }
   }
 
+  "An EmptyData source" should {
+    "read from empty source and write to output without errors" in {
+      HadoopPlatformJobTest(new EmptyDataJob(_), cluster)
+        .run()
+    }
+  }
+
```

Comment:
as is, this should fail, so I'm surprised that it doesn't :(

TestEmptySource doesn't mix in SuccessFileSource, so it shouldn't be happy with an empty directory. Any idea why this isn't failing?